ui/tutorial-renderer/src/main/res/layout/baked_in_tutorial_tools.xml

```diff
@@ -70,7 +70,11 @@
             android:id="@+id/onboarding_next_button"
             android:layout_width="0dp"
             android:layout_height="wrap_content"
-            android:layout_margin="32dp"
+            android:layout_marginStart="32dp"
+            android:layout_marginLeft="32dp"
+            android:layout_marginEnd="32dp"
+            android:layout_marginRight="32dp"
```

Comment:
in the situation that the left/right margin is identical there is no need to use both left/right and start/end. it makes layouts easier to digest if you leave off one of those sets when not necessary.